openstack/dds/v3/instances/requests.go

```diff
@@ -229,6 +232,10 @@ type PortOpts struct {
 	Port int `json:"port"`
 }
 
+type EnabledOpts struct {
+	Enabled bool `json:"enabled"`
```

Comment:
it's better to use a pointer
```suggestion
	Enabled *bool `json:"enabled,omitempty"`
```

Reply:
done